bundles/org.openhab.binding.groupepsa/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,559 @@
+				<default>F2kK7lC5kF5qN7tM0wT8kE3cW1dP0wC5pI6vC0sQ5iP5cN8cJ8</default>
+				<label>Client Secret</label>
+				<description>The client secret for API access: can normally left at the default value. (see:
+					https://developer.groupe-psa.io/webapi/b2c/quickstart/connect/#article)</description>
+			</parameter>
+		</config-description>
+
+	</bridge-type>
+
+	<thing-type id="vehicle">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+
+		<label>Groupe PSA Car</label>
+		<description>Car connected via Groupe PSA (Peugeot, Citroen, Vauxhall, Opel, DS) Bridge</description>
+
+		<channel-groups>
+			<channel-group id="battery" typeId="battery"/>
+			<channel-group id="doors" typeId="doors"/>
+			<channel-group id="environment" typeId="environment"/>
+			<channel-group id="motion" typeId="motion"/>
+			<channel-group id="position" typeId="position"/>
+			<channel-group id="various" typeId="various"/>
+			<channel-group id="fuel" typeId="fuel"/>
+			<channel-group id="electric" typeId="electric"/>
+		</channel-groups>
+
+		<properties>
+			<property name="vendor">N/A</property>
+			<property name="model">N/A</property>
+			<property name="vin">N/A</property>
+		</properties>
+
+		<representation-property>id</representation-property>
+
+		<config-description>
+			<parameter name="id" type="text" required="true">
+				<label>ID</label>
+				<description>Vehicle API ID (obtained automatically during discovery, not equal to the VIN)</description>
+			</parameter>
+			<parameter name="pollingInterval" type="integer" required="false" unit="min">
+				<label>Polling Interval</label>
+				<default>5</default>
+				<description>How often this vehicle should be polled for updated information</description>
+			</parameter>
+			<parameter name="onlineInterval" type="integer" required="false" unit="min">
+				<label>Online Timeout</label>
+				<default>15</default>
+				<description>Timeout when the vehicle is considered offline</description>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-group-type id="battery">
+		<label>Battery</label>
+		<description>Main battery</description>
+		<category>Energy</category>
+		<channels>
+			<channel id="current" typeId="current"/>
+			<channel id="voltage" typeId="level">
+				<label>Battery Level</label>
+				<description>Battery level</description>
+			</channel>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="doors">
+		<label>Doors</label>
+		<description>State of the doors</description>
+		<category>Door</category>
+		<channels>
+			<channel id="locked" typeId="doorLock"/>
+			<channel id="driver" typeId="doorOpen">
+				<label>Driver door</label>
+				<description>Driver door open state</description>
+			</channel>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="environment">
+		<label>Environment</label>
+		<description>Environmental conditions</description>
+		<category>Sun_Clouds</category>
```

Comment:
Are you sure this category exists? It's not listed here https://www.openhab.org/docs/concepts/categories.html#thing-categories

Reply:
It is an existing channel category, I think: https://www.openhab.org/docs/concepts/categories.html#weather